core/Enclave/enclave_hsm.cpp

```diff
@@ -109,19 +111,27 @@ void printf(const char *fmt, ...)
 
 static uint32_t sgx_calc_gcm_data_size(const uint32_t aad_size, const uint32_t plaintext_size)
 {
-    if (aad_size > UINT32_MAX - sizeof(aes_gcm_key_data_t))
+    if (aad_size > UINT32_MAX - sizeof(sgx_aes_gcm_data_ex_t))
```

Comment:
Where is the definition of sgx_aes_gcm_data_ex_t ?

Reply:
in datatypes.h, the same struct as before

Reply:
the structure is defined by ourselves? so not name it as sgx_xxx